test/basic.js

```diff
@@ -64,6 +64,36 @@ describe('request', function(){
         done();
       });
     })
+
+    it('is optional with a promise', function() {
```

Comment:
I think you need to take a `done()` parameter here or mocha will consider the test passed before all the `assert` calls are reached.